server/controllers/student/get-questions.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Response } from 'express';
+import { getQuestionsQuery, checkUserAttendQuizQuery } from '../../database/queries';
+import { UserAuth } from '../../interfaces';
+import { CustomError } from '../../errors';
+
+export default async (req:UserAuth, res:Response, next:NextFunction) => {
+  const { params: { quizId } } = req;
+  const { userId } = req.user;
+
+  try {
+    const { rows } = await getQuestionsQuery(quizId);
```

Comment:
Why did you send a request to the database to bring the questions while you are not sure yet if you will use them or not because the student took the test before?